package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "git-land",
+  "version": "0.0.0",
+  "description": "Merges a pull request or topic branch via rebasing to avoid a merge commit.",
+  "main": "git-land",
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/bazaarvoice/git-land.git"
+  },
+  "keywords": [
+    "git",
+    "cli"
+  ],
+  "bin": {
+    "git-land": "git-land"
+  },
+  "author": "Bazaarvoice",
+  "license": "BV Apache 2.0 License",
```

Comment:
According to https://docs.npmjs.com/files/package.json#license, the right thing for us to do is `"license" : "LicenseRef-LICENSE"` and rename `LICENSE.md` to `LICENSE`.